noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable.nr

```diff
@@ -182,17 +180,15 @@ where
     }
 }
 
-impl<T, let INITIAL_DELAY: u32, let N: u32> SharedMutable<T, INITIAL_DELAY, UnconstrainedContext>
+impl<T, let INITIAL_DELAY: u32, let M: u32> SharedMutable<T, INITIAL_DELAY, UnconstrainedContext>
```

Comment:
T now represents the whole WithHash thing and not just svc so I change N to M to keep it consistent with the rest of this file.

Reply:
I don't think so, in the declaration above you wrote 

```
SharedMutable<T, INITIAL_DELAY, Context>
where
WithHash<SharedMutableValues<T, INITIAL_DELAY>, _>: Packable<M
```

so T is not packed into M, WithHash<SharedMutableValues<T is packed into M.

You also do `SharedMutableValues<T>` below. `T` is whatever we're storing.

Reply:
You are right. Good catch. Reverted the change in https://github.com/AztecProtocol/aztec-packages/pull/11817/commits/c6329b7c3476c22c357ca6de7d113fdf409e42d6